types/src/main/kotlin/gg/skytils/hypixel/types/skyblock/member.kt

```diff
@@ -18,7 +18,9 @@ data class Member(
     val slayer: SlayerData = SlayerData(),
     @SerialName("jacobs_contests")
     val jacob: JacobData = JacobData(),
-    val trophy_fish: TrophyFishData = TrophyFishData()
+    val trophy_fish: TrophyFishData = TrophyFishData(),
+    @SerialName("item_data")
+    val itemData: ItemData = ItemData()
```

Comment:
While camel case is the preferred format for jvm languages, we've used snake case for the rest of the api so it would make sense to keep this consistent.